### DISCUSSION TITLE
Обход блокировок на MacOs (в приоритете Discord)

### DISCUSSION THREAD

**Discussion by MycroftBrooks** [bol-van/zapret#826] [Q&A]
Всем привет. Как я понял на MacOS блокировку Discord не обойти из-за отсутствия dvtws, получиться ли пользоваться дискордом если похимичить на роутере? Есть да, есть ли какой-то мануал для чайника как поставить zapret на роутер МГТС?

**Comment by MycroftBrooks** [comment]
Если решение на роутере не поможет, может есть какая-то альтернатива, есть свой сервак и там развернут Outline, но у Outline как понимаю нельзя на стороне клиента фильровать отдельные домены.

**Reply by Savvy08** [reply]
можно развернуть свой сервер и использовать безопасное соединение для работы дискорда, но как я знаю МГТС дает возможность в некоторых регионах пользоваться дисководом и ютубом
<img width="1438" alt="Снимок экрана 2024-12-26 в 00 56 25" src="https://github.com/user-attachments/assets/7c5fdea2-29cc-4997-8ef3-2569633488e4" />
на данный момент у меня все работает с помощью сервера, кроме текстового чата и голосовой тоже очень хорошо работает, если ты сделаешь туннелирование на определенное приложение, то все будет гуд. в моем случае у меня macOS 13, выше не обновлял

**Reply by MycroftBrooks** [reply]
@Kwertyis Через сторонний сервер выглядит интересным, но хотелось бы подружить связку роутер + zapret (купил отдельный роутер и поставил МГТС роутер как мост). Пока не осилил сделать такую связку, думаю развернуть на сервере впн для дискорда и тунелить чисто дискорд и может ютуб. Есть ли какой-то гайд как это сделать? Также заметил что можно просто тунелировать соединение с дискордом через WARP, но пока не разобрался как это сделать.

**Reply by brnsky** [reply]
> можно развернуть свой сервер и использовать безопасное соединение для работы дискорда, но как я знаю МГТС дает возможность в некоторых регионах пользоваться дисководом и ютубом <img alt="Снимок экрана 2024-12-26 в 00 56 25" width="1438" src="https://private-user-images.githubusercontent.com/86265748/398601665-7c5fdea2-29cc-4997-8ef3-2569633488e4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY0Mzg0MTUsIm5iZiI6MTczNjQzODExNSwicGF0aCI6Ii84NjI2NTc0OC8zOTg2MDE2NjUtN2M1ZmRlYTItMjljYy00OTk3LThlZjMtMjU2OTYzMzQ4OGU0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTA5VDE1NTUxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUyMGRhYjVmMjFhOTZhYTY5ZmNlNzlhNGMwMDM4ZmFkZTZmNTBhY2JjOTJjYzg0M2M1OWRlYTRjMzc4NmY3YmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.jf37MbmkiX9qLILOn7xEq3g3ICiavOVPKZujDvaFHD0"> на данный момент у меня все работает с помощью сервера, кроме текстового чата и голосовой тоже очень хорошо работает, если ты сделаешь туннелирование на определенное приложение, то все будет гуд. в моем случае у меня macOS 13, выше не обновлял

а через что туннелишь дискорд (через какой-то серв, где хостинг брал)? я пока особо удобной альтернативы не нашел, можешь рассказать подробнее, пожалуйста?
